_topic_map.yml

```diff
@@ -478,11 +480,11 @@ Topics:
 - Name: Preparing to update to OKD 4.9
   File: updating-cluster-prepare
   Distros: openshift-origin
-- Name: Updating a cluster between minor versions
+- Name: Updating a cluster within a minor version using the web console
   File: updating-cluster-between-minor
-- Name: Updating a cluster within a minor version from the web console
-  File: updating-cluster
-- Name: Updating a cluster within a minor version by using the CLI
+# - Name: Updating a cluster within a minor version from the web console
+#  File: updating-cluster
```

Comment:
You are are not deleting the files, just hiding the old topic name. Are you planing to delete in future?

Reply:
@LalatenduMohanty Will delete in the future. I was having linking issues and wanted to make sure that it would build without any errors.